src/main/java/duke/Type.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+/**
+ * Enum class for types of events.
+ */
+public enum Type {
```

Comment:
I think this naming might be a bit confusing if I have not seen the rest of your code, because you are referring to the type of `Task`, so maybe `TaskType` would be more suitable?